shared/naturalcrit/markdown.js

```diff
@@ -2,8 +2,13 @@
 const _ = require('lodash');
 const Marked = require('marked');
 const MarkedExtendedTables = require('marked-extended-tables');
+const MarkedMermaid = require('marked-mermaid');
 const renderer = new Marked.Renderer();
 
+const pageElem = global.document
+	? global.document.querySelector('.page')
+	: null;
```

Comment:
Not a big fan of adding a DOM selector here in the `markdown.js` file, but we need this for:
A. Calculating and passing the correct width constraints to Mermaid (used here)
B. The container to add the graph to (optional, but improves graph size calculations)